ASTProcessor/build.gradle.kts

```diff
@@ -19,10 +19,18 @@ repositories {
 dependencies {
 	implementation(libs.springboot.starter.web)
 	implementation(libs.springdoc.openapi)
+	implementation(libs.antlr.runtime)
+	compileOnly(libs.lombok)
+	annotationProcessor(libs.lombok)
 	developmentOnly(libs.springboot.devtools)
 	testImplementation(libs.springboot.starter.test)
 	testRuntimeOnly(libs.junit.launcher)
 	antlr(libs.antlr)
+	testImplementation(libs.logback)
+	compileOnly(libs.lombok)
```

Comment:
You are calling it twice, just call it once either here or above. It would be better if you put it in order, or why is it repeated?

Reply:
Both are required for Lombok to function, one at compile time and also to generate additional code based on the annotations.

Reply:
I see that they are the same and it would not affect anything if we remove it since it is being called the same. Don't be confused with the tests you posted.

![image](https://github.com/user-attachments/assets/48343502-b3da-45ba-9854-20a1f999b11a)

Reply:
ok, now I understand. Done